### PR TITLE
Completed javascript-conditonals exercise

### PR DESCRIPTION
**Pull request by nimcastano**
Screenshot 2023-03-22 at 2 23 45 PM
Screenshot 2023-03-22 at 2 35 55 PM
Screenshot 2023-03-22 at 2 34 40 PM
Screenshot 2023-03-22 at 2 29 04 PM

### CODE REVIEW

File: javascript-conditionals/main.js [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+const isEven = (num) => {
+  return (num % 2 === 0);
+};
+
+const startsWithJ = (str) => {
+  return (str[0] === 'J');
+};
+
+const isOldEnoughToDrink = (person) => {
+  return (person.age > 20);
```
Comment (shawnkost) [inline javascript-conditionals/main.js:25]: What's the legal drinking age 👮🏻 ? We're talking about **legal** age, not when you started drinking 😉

File: javascript-conditionals/main.js [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+
+const isOldEnoughToDrinkAndDrive = (person) => {
+  return false;
+};
+
+const categorizeAcidity = (pH) => {
+  if (pH === 7) {
+    return 'neutral';
+  } else if (pH < 7 && pH > 0) {
+    return 'acid';
```
Comment (shawnkost) [inline javascript-conditionals/main.js:40]: What happens if `pH` is 0? 0 would still be considered acid

File: javascript-conditionals/main.js [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+  } else if (pH < 14 && pH > 7) {
+    return 'base';
```
Comment (shawnkost) [inline javascript-conditionals/main.js:42]: What happens if `ph` is 14? 14 would still be considered base

File: javascript-conditionals/main.js [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+
+const introduceWarnerBro = (name) => {
+  if (name === 'yakko' || name === 'wakko') {
+    return "We're the warner brothers!";
+  } else if (name === 'dot') {
+    return "I'm cute~";
+  } else {
+    return 'Goodnight Everybody!';
+  }
+};
```
Comment (shawnkost) [inline javascript-conditionals/main.js:56]: This needs to be completed using a `switch` statement.

**Review by shawnkost — CHANGES_REQUESTED** [review]
A few changes requested here!

File: javascript-conditionals/main.js (hunk 3 above)
Comment (nimcastano) [reply javascript-conditionals/main.js]: Thank you! Constantly struggling with edge cases lol

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**